extensions/wikia/WikiFactory/WikiFactory.php

```diff
@@ -1168,7 +1168,7 @@ static public function IDtoDB( $city_id, $master = false ) {
 	 * @return string changed host or $default
 	 */
 	public static function getCurrentStagingHost( $dbName='', $default='', $host = null) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Reduce the number of returns of this function 4, down to the maximum allowed 3. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1142)